tutor/resources/styles/components/new-course.less

```diff
@@ -234,29 +226,11 @@
       background-position: left center;
     }
 
-    [data-copy-or-not],
-    [data-new-or-copy] {
-      &::before {
-        background-position: center;
```

Comment:
this is still needed for `[data-new-or-copy] `
![screen shot 2016-12-13 at 11 50 45 am](https://cloud.githubusercontent.com/assets/2483873/21152053/66ac632e-c12a-11e6-9765-5e4f3ab5a124.png)